tomcat/src/main/java/org/apache/coyote/http11/SessionManager.java

```diff
@@ -1,13 +1,13 @@
 package org.apache.coyote.http11;
 
-import java.util.HashMap;
 import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
 
-public class SessionManger {
+public class SessionManager {
 
-    private static final Map<String, Session> SESSIONS = new HashMap<>();
+    private static final Map<String, Session> SESSIONS = new ConcurrentHashMap<>();
```

Comment:
```suggestion
    private static final Map<String, Session> sessions = new ConcurrentHashMap<>();
```

이건 저도 몰랐는데 인스턴스는 상수가 아니라고 하네요..!

[https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names)

Reply:
아 그렇네요 저도 이번에 처음 알았습니다 꼼꼼한 리뷰 감사합니다!